01-hello-world/helloWorld.js

```diff
@@ -5,6 +5,7 @@
 function sayHello(name)
 {
     // Code goes here
```

Comment:
Since you put the code in, there's no reason for this comment to continue to exist.